src/Servers/Kestrel/shared/TransportConnection.Generated.cs

```diff
@@ -19,6 +19,7 @@ internal partial class TransportConnection : IFeatureCollection
         private object? _currentIConnectionItemsFeature;
         private object? _currentIMemoryPoolFeature;
         private object? _currentIConnectionLifetimeFeature;
+        private object? _currentIConnectionSocketFeature;
```

Comment:
This is a generated file. Was it updated manually? It should be updated by starting here:
https://github.com/dotnet/aspnetcore/blob/main/src/Servers/Kestrel/tools/CodeGenerator/TransportConnectionFeatureCollection.cs
And then running that project to regenerate this file.

Reply:
Hi @Tratcher , Thanks for the response! Based on your comment, I have generated that file using the CodeGenerator tool! Can you please check now, is that correct? 

If yes, based on my changes, what should my next step!

Thanks again! :)